reduction/lr_reduction/workflow.py

```diff
@@ -12,7 +12,7 @@
 
 def reduce(ws, template_file, output_dir, average_overlap=False,
            q_summing=False, bck_in_q=False, is_live=False,
-           functional_background=False):
+           functional_background=False, dead_time=False):
```

Comment:
is the optional argument paralyzable of EventReflectivity.__init__ left out here on purpose?